### PR TITLE
Ferie/ferieind 20 lederoverblik godkend frontend

### PR DESCRIPTION
**Pull request by Ranthony67**

### CODE REVIEW

File: Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html [hunk 1]
```diff
@@ -1,77 +1,63 @@
...
-                <div class="col-lg-1">September</div>
-                <div class="col-lg-1">Oktober</div>
-                <div class="col-lg-1">November</div>
-                <div class="col-lg-1">December</div>
             </div>
         </div>
+    </div>
+</div>
+
+<!--<script>
```
Comment (snebjorn) [inline Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html]: Removed unused code

File: Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html [hunk 2]
```diff
@@ -1,77 +1,63 @@
...
-            </div>
-        </div>
+    <span class="k-widget k-dropdown k-header" unselectable="on" role="listbox" aria-haspopup="true" aria-expanded="false" tabindex="0" aria-owns="" aria-disabled="false" aria-readonly="false" aria-busy="false" style="margin-left: 3%">
+        <span unselectable="on" class="k-dropdown-wrap k-state-default k-state-hover">
+            <span unselectable="on" class="k-input">Pending</span>
+            <span unselectable="on" class="k-select">
+                <span unselectable="on" class="k-icon k-i-arrow-s">select</span>
+            </span>
+        </span>
+        <select data-bind="value:status" data-role="dropdownlist" style="display: none;">
```
Comment (snebjorn) [inline Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html]: style="display: none;" is ngHide

File: Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html [hunk 3]
```diff
@@ -1,77 +1,63 @@
...
-        <div class="row">
-            <div class="col-lg-12">
-                <div kendo-scheduler culture="'da-DK'" k-options="arc.SchdulerOptions">
-                    <span k-event-template class='custom-event'>{{dataItem.title}}</span>
-                    <div k-all-day-event-template class='custom-all-day-event'>{{dataItem.title}}</div>
-                </div>
-
-            </div>
-        </div>
+    <span class="k-widget k-dropdown k-header" unselectable="on" role="listbox" aria-haspopup="true" aria-expanded="false" tabindex="0" aria-owns="" aria-disabled="false" aria-readonly="false" aria-busy="false" style="margin-left: 3%">
```
Comment (snebjorn) [inline Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html]: Are k-widget k-dropdown k-header css classes or angular directives?

File: Presentation.Web/App/Vacation/ApproveReports/ApproveVacationController.ts [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+            this.schedulerOptions = {
+                views: [
+                    {
+                        type: "timelineMonth",
+                        title: "Måned",
+                        minorTickCount: 1,
+                        columnWidth: 25,
+                        dateHeaderTemplate: "<span class='k-link k-nav-day' style=>#=kendo.toString(date, 'dd')#</span>",
+                        //slotTemplate: "<div style='background:#=changeColorforWeekend(date)#; height: 100%;width: 100%; padding: 0;'></div>",
```
Comment (snebjorn) [inline Presentation.Web/App/Vacation/ApproveReports/ApproveVacationController.ts]: remove

File: Presentation.Web/App/Vacation/ApproveReports/ApproveVacationController.ts [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+            "VacationReport",
+            "NotificationService",
+            "$http",
+            "$modal",
+            "moment",
+            "$state"
+        ];
+
+        constructor(private $scope, private $rootScope, private VacationReport, private NotificationService, private $http: ng.IHttpService, private $modal, private moment: moment.MomentStatic, private $state: ng.ui.IStateService) {
+            this._currentUser = $scope.CurrentUser; //TODO: Remove?
```
Comment (snebjorn) [inline Presentation.Web/App/Vacation/ApproveReports/ApproveVacationController.ts]: resolve todo

File: Presentation.Web/Views/Shared/_Layout.cshtml [hunk 6]
```diff
@@ -59,11 +55,10 @@
     <!-- ENABLED MODULES FROM WEB.CONFIG-->
     <script>
         angular.module("app").constant("EnabledApplications", "@System.Configuration.ConfigurationManager.AppSettings["ENABLED_APPLICATIONS"]");
-    </script>
-
-
-
+    </script>
```
Comment (snebjorn) [inline Presentation.Web/Views/Shared/_Layout.cshtml]: No reason to close the script tag just to open it again on the next line

File: Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html (hunk 3 above)
Comment (Ranthony67) [inline Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html]: kendo css classes.

File: Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html [hunk 7]
```diff
@@ -48,7 +38,7 @@
                 <span unselectable="on" class="k-icon k-i-arrow-s">select</span>
             </span>
         </span>
-        <select data-bind="value:status" data-role="dropdownlist" style="display: none;">
+        <select class="ngHide" data-bind="value:status" data-role="dropdownlist">
```
Comment (snebjorn) [inline Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html]: Not quite https://docs.angularjs.org/api/ng/directive/ngHide
Also what is going on here `data-bind="value:status"` that looks like a knockout binding.
Furthermore if the select is always hidden why is it needed?

File: Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html [hunk 8]
```diff
@@ -1,77 +1,51 @@
...
-            </div>
-        </div>
+    <span class="k-widget k-dropdown k-header" unselectable="on" role="listbox" aria-haspopup="true" aria-expanded="false" tabindex="0" aria-owns="" aria-disabled="false" aria-readonly="false" aria-busy="false" style="margin-left: 3%">
+        <span unselectable="on" class="k-dropdown-wrap k-state-default k-state-hover">
+            <span unselectable="on" class="k-input">Pending</span>
+            <span unselectable="on" class="k-select">
+                <span unselectable="on" class="k-icon k-i-arrow-s">select</span>
+            </span>
+        </span>
+        <select class="ngHide" data-bind="value:status" data-role="dropdownlist">
```
Comment (snebjorn) [inline Presentation.Web/App/Vacation/ApproveReports/ApproveReportsView.html]: Found this little piggy

File: Presentation.Web/App/Vacation/ApproveReports/ApproveVacationController.ts [hunk 9]
```diff
@@ -0,0 +1,202 @@
...
+            "$rootScope",
+            "VacationReport",
+            "NotificationService",
+            "$http",
+            "$modal",
+            "moment",
+            "$state"
+        ];
+
+        constructor(private $scope, private $rootScope, private VacationReport, private NotificationService, private $http: ng.IHttpService, private $modal, private moment: moment.MomentStatic, private $state: ng.ui.IStateService) {
```
Comment (snebjorn) [inline Presentation.Web/App/Vacation/ApproveReports/ApproveVacationController.ts]: Missing type on $modal

**Pull request merged**